operations/jsonnet/microservices/querier.libsonnet

```diff
@@ -24,7 +24,10 @@
     ]) +
     container.withVolumeMounts([
       volumeMount.new(tempo_config_volume, '/conf'),
-    ]),
+    ]) +
+    container.withEnvMap({
+      JAEGER_DISABLED: 'true',
```

Comment:
I don't think it's `tempo-querier` producing these traces, it looks like they are coming from the `tempo-query`, i.e. the Jaeger container.

Reply:
Thanks for pointing that out @bombsimon!